src/components/Frame/Frame.component.jsx

```diff
@@ -0,0 +1,149 @@
+import React from 'react'
+import styled from 'styled-components'
+import * as IconName from 'react-icons/bi'
+import * as AiIcons from 'react-icons/ai'
+import PropTypes from 'prop-types'
+import Device from '../../styles/Device'
+
+const Container = styled.div`
+    display: flex;
+    align-items: flex-start;
+    flex-direction: column;
+    padding-right: 24px;
+`
+const FrameVideo = styled.iframe`
+    width: 100%;
+    height: auto;
+
+    @media ${Device.mobileS} {
+        min-height: 180px;
+    }
+    @media ${Device.mobileM} {
+        min-height: 280px;
+    }
+    @media ${Device.mobileL} {
+        min-height: 320px;
+    }
+    @media ${Device.tablet} {
+        min-height: 360px;
+    }
+    @media ${Device.laptop} {
+        min-height: 480px;
+    }
+`
+const CardDetail = styled.div`
+    display: flex;
+
+    flex-direction: column;
+    padding-top: 20px;
+    padding-bottom: 8px;
+    h1 {
+        font-size: 1.5em;
+        padding-bottom: 8px;
+    }
+    h2 {
+        font-size: 1em;
+        padding-bottom: 8px;
+    }
+`
+const Info = styled.div`
+    display: flex;
+    flex-direction: row;
+    padding-top: 20px;
+    padding-bottom: 8px;
+    justify-content: space-between;
+    color: #737373;
+`
+const Icon = styled.div`
+    font-size: 1.5rem;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    color: #737373;
+`
+const IconText = styled.div`
+    font-size: 1.2rem;
+`
+const Channel = styled.div`
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    svg {
+        margin-bottom: 5px;
+        padding: 5px;
+    }
+`
+const Separator = styled.hr`
+    margin-top: 24px;
+    margin-bottom: 24px;
+`
+
+function Frame({ video }) {
+    return (
+        <Container>
+            {video && video.snippet && video.snippet.localized ? (
```

Comment:
**Optional**: 

A cool JS trick is [Optional Chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) so you can write 
```js
{video?.snippet?.localized ? (
```
to achieve the same result

Reply:
Cool and awesome Js trick, from now on i'm going to use it :)